<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recharge_tool_dialog</class>
 <widget class="QDialog" name="recharge_tool_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择组件</string>
  </property>
  <widget class="QPushButton" name="choice_button_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_label_1">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="choice_button_2">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>90</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="choice_button_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_label_3">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="choice_button_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_label_4">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="choice_button_5">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>190</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="choice_button_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_label_6">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>150</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="component_list">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>已选组件：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button_1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>聚焦光束</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>探测手杖</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>瘫痪魔杖</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
